app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,17 @@
 <% end %>
 
 <% content_for :content do %>
+  <div class="search">
+    <form action="<%=root_path%>", method="get">
```

Comment:
Having a more explicit `guides_path` here would be safer, because the root_url can change to point to another controller action.